src/hooks/core/chat/useDMMessage.ts

```diff
@@ -14,7 +14,7 @@ type ChannelCategory = 'SIGNED' | 'ANONYMOUS';
 const useDMMessage = () => {
   const {localDb, refresh} = useLocalDatabaseHook();
   const {signedProfileId, anonProfileId} = useUserAuthHook();
-  const {goToChatScreen} = useChatUtilsHook();
+  const {goToChatScreen} = useChatUtilsHook('SIGNED');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `useChatUtilsHook()` to `useChatUtilsHook('SIGNED')` seems to be a significant one. It's important to ensure that this doesn't break any existing functionality where the default parameter was relied upon. If 'SIGNED' is now always required, it might be worth updating the function definition to reflect this, or at least adding a comment explaining why this value is hardcoded here.
```diff
-  const {goToChatScreen} = useChatUtilsHook();
+  const {goToChatScreen} = useChatUtilsHook('SIGNED');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->